test/test_decomp.py

```diff
@@ -167,7 +167,10 @@ def op_assert_ref(test_case, op, test_dtype, i, orig, decomp, ref, args, kwargs)
         (torch.float16, torch.ops.aten._native_batch_norm_legit.no_stats): 1e-5,
         (torch.bfloat16, torch.ops.aten.linalg_vector_norm.default): 1e-4,
         (torch.float16, torch.ops.aten.linalg_vector_norm.default): 1e-4,
+        (torch.bfloat16, torch.ops.aten.var_mean.correction): 5e-7,
```

Comment:
why does tolerance change here?

Reply:
`aten.var_mean` uses a different algorithm from `aten.mean` which ends up being slightly more precise.  `5e-7` is still incredibly good for half-precision though. The default `rtol` for `torch.testing` is `1e-5`.